### ISSUE TITLE
phpcbf: array in multiline arguments

### ISSUE THREAD

**Issue by komex** [squizlabs/PHP_CodeSniffer#829]
phpcbf fix code incorrect from
```php
$this->listeners[] = $events->getSharedManager()->attach('Zend\Mvc\Application', MvcEvent::EVENT_DISPATCH, [$this, 'selectLayout'], 100);
```
to
```php
$this->listeners[] = $events->getSharedManager()->attach(
    'Zend\Mvc\Application',
    MvcEvent::EVENT_DISPATCH,
    [$this,
    'selectLayout'],
    100
);
```

**Comment by gsherwood** [comment]
None of the included standards would report any errors, or attempt to fix anything, on the code sample you've provided. They only give warnings about the line length.

Are you using an included standard or a custom standard?

You could also try running PHPCS instead of PHPCBF and passing `--report=diff` on the command line. This will show you the diff that PHPCBF would apply. Using `--report=diff -vv` instead will give you information about which sniff is making each change.

**Comment by komex** [comment]
@gsherwood, sorry, first code fragment is invalid. Correct is:
```php
$this->listeners[] = $events->getSharedManager()->attach(
        'Zend\Mvc\Application', MvcEvent::EVENT_DISPATCH, [$this, 'selectLayout'], 100
);
```

PHPCS with `--report=diff -vv` shows:
```
*** START FILE FIXING ***
	E: [Line 60] Only one argument is allowed per line in a multi-line function call (PSR2.Methods.FunctionCallSignature.MultipleArguments)
	=> Changeset started by PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 473)
		Q: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 479) replaced token 317 (T_WHITESPACE) "·MvcEvent" => "MvcEvent"
		Q: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 485) replaced token 318 (T_STRING) "MvcEvent" => "\n············MvcEvent"
		A: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 486) replaced token 317 (T_WHITESPACE) "·MvcEvent" => "MvcEvent"
		A: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 486) replaced token 318 (T_STRING) "MvcEvent" => "\n············MvcEvent"
	=> Changeset ended: 2 changes applied
	E: [Line 60] Only one argument is allowed per line in a multi-line function call (PSR2.Methods.FunctionCallSignature.MultipleArguments)
	=> Changeset started by PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 473)
		Q: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 479) replaced token 322 (T_WHITESPACE) "·[" => "["
		Q: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 485) replaced token 323 (T_OPEN_SHORT_ARRAY) "[" => "\n············["
		A: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 486) replaced token 322 (T_WHITESPACE) "·[" => "["
		A: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 486) replaced token 323 (T_OPEN_SHORT_ARRAY) "[" => "\n············["
	=> Changeset ended: 2 changes applied
	E: [Line 60] Only one argument is allowed per line in a multi-line function call (PSR2.Methods.FunctionCallSignature.MultipleArguments)
	=> Changeset started by PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 473)
		Q: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 479) replaced token 326 (T_WHITESPACE) "·'selectLayout'" => "'selectLayout'"
		Q: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 485) replaced token 327 (T_CONSTANT_ENCAPSED_STRING) "'selectLayout'" => "\n············'selectLayout'"
		A: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 486) replaced token 326 (T_WHITESPACE) "·'selectLayout'" => "'selectLayout'"
		A: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 486) replaced token 327 (T_CONSTANT_ENCAPSED_STRING) "'selectLayout'" => "\n············'selectLayout'"
	=> Changeset ended: 2 changes applied
	E: [Line 60] Only one argument is allowed per line in a multi-line function call (PSR2.Methods.FunctionCallSignature.MultipleArguments)
	=> Changeset started by PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 473)
		Q: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 479) replaced token 330 (T_WHITESPACE) "·100" => "100"
		Q: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 485) replaced token 331 (T_LNUMBER) "100" => "\n············100"
		A: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 486) replaced token 330 (T_WHITESPACE) "·100" => "100"
		A: PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 486) replaced token 331 (T_LNUMBER) "100" => "\n············100"
	=> Changeset ended: 2 changes applied
	* fixed 8 violations, starting loop 2 *
	*** END FILE FIXING ***
```

**Comment by gsherwood** [comment]
This is fixed in the master branch already, although I'm not 100% sure which commit fixed it. Here is what I get when I check the code you supplied:
```
FILE: temp.php
------------------------------------------------------------------------------------------------
FOUND 4 ERRORS AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 3 | ERROR | [x] Multi-line function call not indented correctly; expected 4 spaces but found 8
 3 | ERROR | [x] Only one argument is allowed per line in a multi-line function call
 3 | ERROR | [x] Only one argument is allowed per line in a multi-line function call
 3 | ERROR | [x] Only one argument is allowed per line in a multi-line function call
------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 4 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------
```
And when I use PHPCBF, the code is fixed into this:
```php
<?php
$this->listeners[] = $events->getSharedManager()->attach(
    'Zend\Mvc\Application',
    MvcEvent::EVENT_DISPATCH,
    [$this, 'selectLayout'],
    100
);
```
All that looks good to me now. Thanks for reporting the issue.
